dependency_support/org_theopenroadproject_asap7/bundled.BUILD.bazel

```diff
@@ -59,6 +59,7 @@ open_road_pdk_configuration(
         "M7": "0.5",
     },
     global_routing_signal_layers = "M2-M7",
+    klayout_tech_file = "@rules_hdl//dependency_support/org_theopenroadproject_asap7:asap7.lyt",
```

Comment:
q: no fill config by default in ASAP7?

Reply:
Looking at the flow examples in OpenROAD-flow-scripts - no, there is no `density_fill` stage in the flow targeting ASAP7 so there is no need for specifying the config file